Orchestrator.py

```diff
@@ -438,4 +390,69 @@ def _computeCellNeighbours(self, cx, cy):
             (cx-cellSize, cy+cellSize), (cx+cellSize, cy-cellSize)
         ]
 
+        return returnVal
+
+    def _computeCellCorners(self, x, y):
+
+        (cxmin, cymin)    = self._xy2cell(x, y)
+        (cxmax, cymax)    = (cxmin + self.MINFEATURESIZE/2, cymin + self.MINFEATURESIZE/2)
+
+        topLeftCorner     = (cxmin, cymin)
+        topRightCorner    = (cxmax, cymin)
+        bottomLeftCorner  = (cxmin, cymax)
+        bottomRightCorner = (cxmax, cymax)
+
+        return [topLeftCorner, topRightCorner, bottomLeftCorner, bottomRightCorner]
+
+    def _checkForCornerFrontier(self, frontier):
```

Comment:
add a comment which specifies format of "frontier" (most probably not the right name)